filer/templates/admin/filer/folder/directory_table_list.html

```diff
@@ -139,7 +139,7 @@
                             </td>
                             <td class="column-action">
                                 {% if file.canonical_url %}
-                                    <a href="{% if 'svg' in file.mime_type %}{% url 'admin:filer_image_expand_view' file.pk %}{% else %}{{ file.canonical_url }}{% endif %}"
+                                    <a href="{% if 'svg' in file.mime_type %}{% url 'admin:filer_image_expand' file.pk %}{% else %}{{ file.canonical_url }}{% endif %}"
```

Comment:
Well spotted! 
I would like to check if this also works for a custom image model.